static/js/utils/disaster_event_map_utils.tsx

```diff
@@ -142,6 +143,36 @@ export function fetchDateList(
   });
 }
 
+/**
+ * Get a user-friendly info card title to use for each event.
+ *
+ * Titles follow the format `<EventType>: <Name>`.
+ *            Ex: "Storm: Ian"
+ * This also converts names that are "<EventType> that started on <Date> at
+ * <S2Cell>" into "Unnamed" for the title.
```

Comment:
Not a fan of using Unnamed. We should include place name or something better. If we go with this, all events of certain types will be unnamed.

Do we also need the event type in the title? We can add the type as an annotation in the card.

This should go in a event --> name util that we can use across tiles (or pages) so the events show up consistently.